ndr_fasttransfer.c

```diff
@@ -261,6 +261,43 @@ static int ndr_parse_property(TALLOC_CTX *mem_ctx, struct ndr_print *ndr,
 	return -1;
 }
 
+/* Parse namedPropInfo element which includes:
+     PytpGuid ((%x00 PytpInteger32) | (%x01 PtypString))
+   to set the name and kind of the parsed named property
+*/
+static int ndr_parse_namedproperty(TALLOC_CTX *mem_ctx, struct ndr_print *ndr,
+				   struct ndr_pull *ndr_pull, uint32_t element,
+				   char **named_prop)
+{
+	const char  *name;
+	struct GUID PytpGuid;
+	uint8_t	    type;
+	uint32_t    dispid;
+
+	if (!named_prop) {
+		return -1;
+	}
+
+	NDR_CHECK(ndr_pull_GUID(ndr_pull, NDR_SCALARS, &PytpGuid));
+
+	NDR_CHECK(ndr_pull_uint8(ndr_pull, NDR_SCALARS, &type));
+	if (type == 0) {
+		NDR_CHECK(ndr_pull_uint32(ndr_pull, NDR_SCALARS, &dispid));
+		*named_prop = talloc_asprintf_append(*named_prop, " (LID 0x%08x)", dispid);
+	} else if (type == 1) {
```

Comment:
I suppose type is to specify `ulkind` value. Hence, I'd rather like to see MNID_ID and MNID_STRING in place of int values.